src/agent/debuglet.ts

```diff
@@ -175,6 +179,9 @@ export class Debuglet extends EventEmitter {
 
     /** @private {Object.<string, Boolean>} */
     this.completedBreakpointMap_ = {};
+
+    /** @private {Array<number>} */
+    this.promiseResolvedTimestamp_ = [0, 0];
```

Comment:
Per the Node document, passing a user-defined array to process.hrtime() is an undefined behavior. I think you should initialize this with the result of process.hrtime().

Reply:
I change hrtime to Date.now() and will initialize with Date.now()